Header header

uint16    stepsize #should be 272 * 2 = 544
uint16[]  num_array
float32[] data_a
float32[] data_b